charts/sourcegraph/values.yaml

```diff
@@ -503,7 +503,7 @@ grafana:
 indexedSearch:
   image:
     # -- Docker image tag for the `zoekt-webserver` image
-    defaultTag: 4.1.0@sha256:b82f01effbe0ef407557bbe0f1c83f3bb47f7fab64584532590fae6e125c5e7f
+    defaultTag: 4.1.1@sha256:b82f01effbe0ef407557bbe0f1c83f3bb47f7fab64584532590fae6e125c5e7f
```

Comment:
weirdly these don't seem to change the hash. not sure what that means. maybe it just didn't change.

Reply:
Weird. I'll ask in #dev-chat if that's expected?